tests/integration/targets/aci_l3out_logical_node/tasks/main.yml

```diff
@@ -11,11 +16,11 @@
       validate_certs: '{{ aci_validate_certs | default(false) }}'
       use_ssl: '{{ aci_use_ssl | default(true) }}'
       use_proxy: '{{ aci_use_proxy | default(true) }}'
-      output_level: '{{ aci_output_level | default("info") }}'
+      output_level: debug
```

Comment:
any reason for this change?

Reply:
Not specially, just to be in line with the current way we make the test cases

Reply:
but you're right, we should keep it as it was previously, it was just for the test.